ha/test/integration/health_monitor/test_publisher.py

```diff
@@ -43,12 +43,12 @@
         print("Consuming the action event")
         message_consumer = MessageBus.get_consumer(consumer_id="1",
                             consumer_group='test_publisher',
-                            message_types=[message_type])
+                            message_type=message_type)
```

Comment:
The MessageBus here is from ha.util.message_bus ; and it does not have a get_consumer method
Need to import MessageBus from ha/utils

Reply:
I can see, its imported from ha.util and and MessageBus class has a static method get_consumer().

But, earlier the type was getting send as list and now its str. And actual implementation now accepts str. So, was the MessageBus logic got changed  or this was not captured earlier?

Reply:
Updated.